package.json

```diff
@@ -17,22 +17,23 @@
 		"db:studio:prod": "dotenv -- pnpm --filter db run studio:prod",
 		"proto:compile": "dotenv -- pnpm --filter proto run compile",
 		"clean": "turbo run clean && git clean -xdf .turbo node_modules",
-        "execute": "dotenv -- node scripts/execute.cjs"
+		"execute": "dotenv -- node scripts/execute.cjs"
 	},
 	"devDependencies": {
 		"@energyleaf/tsconfig": "workspace:*",
 		"@total-typescript/ts-reset": "^0.5.1",
 		"jiti": "^1.21.6",
-		"turbo": "^1.13.3"
+		"turbo": "^1.13.4"
 	},
 	"packageManager": "pnpm@9.1.4",
 	"name": "energyleaf",
 	"dependencies": {
-		"@biomejs/biome": "^1.7.3",
+		"@biomejs/biome": "^1.8.1",
 		"@energyleaf/db": "workspace:*",
-		"@vercel/analytics": "^1.2.2",
+		"@vercel/analytics": "^1.3.1",
+		"@vercel/functions": "^1.0.2",
```

Comment:
Sehe ich jetzt erst warum genau sind die im Root installiert und nicht in den jeweiligen packages? 😅